velociraptor/catalogue/registration.py

```diff
@@ -899,6 +930,76 @@ def registration_gas_species_masses(
         raise RegistrationDoesNotMatchError
 
 
+def registration_gas_element_ratios_times_masses(
+    field_path: str, unit_system: VelociraptorUnits
+) -> (unyt.Unit, str, str):
+    """
+    Registers the Fe/H times mass and O/H times mass within apertures
+    """
+
+    unit = unit_system.mass
+
+    # Capture aperture size
+    match_string = "Aperture_([a-zA-Z]*)_aperture_total_gas_([0-9]*)_kpc"
+    regex = cached_regex(match_string)
+
+    match = regex.match(field_path)
+
+    if match:
+        long_species = match.group(1)
+        aperture_size = match.group(2)
+
+        try:
+            short_species = {
+                "OxygenOverHydrogenMasses": "O_over_H",
+                "IronOverHydrogenMasses": "Fe_over_H",
+            }[long_species]
+        except KeyError:
+            raise RegistrationDoesNotMatchError
+
+        full_name = f"{short_species} multiplied by gas mass ({aperture_size} kpc)"
+        snake_case = f"{short_species}_time_mass_{aperture_size}_kpc"
```

Comment:
So this isn't the Oxygen mass, it's the Oxygen mass * hydrogen mass / total mass?

In that case, let's write this as:

```
Oxygen Weighted Gas Mass $(O/H) M_g$ (100 kpc).
```

Reply:
The snake case can then be `oxygen_weighted_mass_100_kpc`

Reply:
It's actually aperture `\sum(mass_gas * X_O / X_H)`

Reply:
How would you call that? The goal is to divide that by `\sum(mass_gas)` in the pipeline to have a mass-weighted O/H.

Reply:
This may sound weird but is it not best then to write it as `(O/H) Gas Mass ($M_{\\rm (O/H)}$)`? As that's what it really measures.

Reply:
If anything, at this stage, it's the (O/H)-weighted gas mass.

Reply:
Would that not be then in 'human' terms the Oxygen Weighted Gas Mass?

Reply:
Oxygen abundance weighted?

Reply:
Sure, `Oxygen Abundance Weighted Gas Mass`

Reply:
Done.